main.tf

```diff
@@ -94,6 +94,15 @@ resource "helm_release" "this" {
     }
   }
 
+  dynamic "set" {
+    for_each = local.iam_role_enabled && var.service_account_role_arn_annotation_enabled ? [module.eks_iam_role.service_account_role_arn] : []
+    content {
+      name  = "serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn"
```

Comment:
Would we ever need to override this ?

Reply:
Hopefully not because it's standard, but the escape hatch is to set `var.service_account_role_arn_annotation_enabled` to false and supply `var.set` or `var.values`.